config/customization.rb.template

```diff
@@ -0,0 +1,2 @@
+ADMIN_EMAIL = "betydb@gmail.com"
```

Comment:
One case where "splitting" is definitely preferable to "lumping" is if you are going to want multiple template files for each aspect since in general the number `n_1 x n_2 x ... x n_m` of template files required by lumping is greater than the `n_1 + n_2 + ... + n_m` template files required for splitting if `m` and the `n_i's` are all >= 2.  But since in our case we only have two `additional_environment` templates and only one `customization` template, we'd actually eliminate one template file by lumping.  (I _had_ considered making per-environment customization files, though, so that the "real" administrator wouldn't get e-mail from developers doing testing, but I tabled that idea for now.)  I still think the two sets of setting are largely orthogonal, however, since _where_ one sends an e-mail is likely to depend on different factors than _how_ one sends e-mail, and also because the latter seems more technical.  But I won't belabor this any further and will wrap the customizations into `additional_environment`.

I would eventually like to fold all the site and instance-dependent settings into a yml or xml file.  Ryan Bates has a Rails-cast on how to put config settings into a yml file.  But I'd probably prefer xml if we're going to want to put anything very complex in there, such as the HTML snippet to use for a footer.